ChronoKeeper/KeeperIdCard.h

```diff
@@ -0,0 +1,95 @@
+#ifndef CHRONOLOG_KEEPER_ID_CARD_H
+#define CHRONOLOG_KEEPER_ID_CARD_H
+
+#include <arpa/inet.h>
+
+#include <ostream>
+
+// this class wrapps ChronoKeeper Process identification 
+// that will be used by all the ChronoLog Processes 
+// to both identofy the Keepr process and create RPC client channels 
+// to send the data to the Keeper Recording service
```

Comment:
Can you maintain one version of a certain header file and put it under include? There are two copies of all the headers in include.

Reply:
./ChronoKeeper/include directory is removed, there's one version of KeeperIdCard.h